code/rdb/datastripe.q

```diff
@@ -73,4 +73,9 @@ initdatastripe:{
 getaccess:{[] `location`table xkey update location:.proc.procname,proctype:.proc.proctype from .ds.access};
 
 // function to update the access table in the gateway. Takes the gateway handle as argument
-updategw:{[h]neg[h](`.ds.updateaccess;getaccess[]);};
+updategw:{[h]
+
+	newtab:getaccess[];
+ 	neg[h](`.ds.updateaccess;newtab);
+
+ 	};
```

Comment:
this seems like a regression? the updated version should work the same, is there a reason to revert it back to this version?

Reply:
looks to be accidentally pushed during testing - I'll sort that one out now